frontend/src/pages/Landing/Payments/LandingPaymentsPage.tsx

```diff
@@ -264,6 +264,7 @@ export const LandingPaymentsPage = (): JSX.Element => {
                       sendOnboardingEmailMutation.isSuccess,
                     placeholder: 'Enter your email',
                   }}
+                  responseMode={FormResponseMode.Email}
```

Comment:
To consider if we can remove `responseMode` from `<EmailField />`. Might not need the fields to know about the `responseMode`, cursory search suggest that it is used to enforce `required` validation

Reply:
Similar for the rest of the fields (i.e., `frontend/src/templates/Field/Dropdown/DropdownField.tsx`)

Reply:
Hmmmm. that's fair. I did think about whether we want to just have a generic `requiredOverride` prop or something similar. Not too sure how to name it, let me think a bit on this.

Reply:
updated to `disableRequiredValidation`